integration-testing/election-manager/cypress/support/read_most_recent_file.ts

```diff
@@ -0,0 +1,13 @@
+import { promises as fs } from 'fs';
+import { join } from 'path';
+
+export async function readMostRecentFile(directoryPath: string) {
```

Comment:
Noticed that we weren't actually using this function in `integration-testing/election-manager/cypress.config.ts` so I pushed a quick patch! https://github.com/votingworks/vxsuite/pull/2189/commits/63d9380d2689b1b20259bf973620264641cde945